hibernate-reactive-core/src/test/java/org/hibernate/reactive/SchemaUpdatePostgreSqlTestBase.java

```diff
@@ -0,0 +1,339 @@
+		updateHbm2ddlConf.addAnnotatedClass( ASimpleNext.class );
+		updateHbm2ddlConf.addAnnotatedClass( AOther.class );
+		updateHbm2ddlConf.addAnnotatedClass( AAnother.class );
+		test(
+				context,
+				setupSessionFactory( updateHbm2ddlConf )
+						.thenCompose( v -> getSessionFactory()
+								.withTransaction( (session, t) -> session.persist( aSimple ) ) )
+						.thenCompose( v1 -> openSession()
+								.thenCompose( s -> s
+										.find( ASimpleNext.class, aSimple.id )
+										.thenAccept( result -> {
+											context.assertNotNull( result );
+											context.assertEquals( aSimple.aValue, result.aValue );
+											context.assertEquals( aSimple.aStringValue, result.aStringValue );
+											context.assertEquals( aSimple.data, result.data );
+											context.assertNotNull( result.aOther );
+											context.assertEquals( aOther.id1, result.aOther.id1 );
+											context.assertEquals( aOther.id2, result.aOther.id2 );
+											context.assertEquals( aOther.anotherString, result.aOther.anotherString );
+											context.assertNotNull( result.aAnother );
+											context.assertEquals( aAnother.description, result.aAnother.description );
+										} )
+										.thenCompose( v -> s.createNativeQuery( indexDefinitionQuery, String.class )
+												.setParameter( 1, "asimple" )
+												.getResultList()
+												.thenAccept( list -> {
+													context.assertEquals(
+															list.get( 0 ),
+															"CREATE UNIQUE INDEX asimple_pkey ON public.asimple USING btree (id)"
+													);
+													context.assertEquals(
+															list.get( 1 ),
+															"CREATE INDEX i_asimple_avalue_astringvalue ON public.asimple USING btree (avalue, astringvalue DESC)"
+													);
+													context.assertEquals(
+															list.get( 2 ),
+															"CREATE INDEX i_asimple_avalue_data ON public.asimple USING btree (avalue DESC, data)"
+													);
+													context.assertEquals(
+															list.get( 3 ),
+															"CREATE UNIQUE INDEX u_asimple_astringvalue ON public.asimple USING btree (astringvalue)"
+													);
+												} )
+										)
+										.thenCompose( v -> s.createNativeQuery( indexDefinitionQuery, String.class )
+												.setParameter( 1, "aother" )
+												.getSingleResult()
+												.thenAccept( result ->
+																	 context.assertEquals(
+																			 result,
+																			 "CREATE UNIQUE INDEX aother_pkey ON public.aother USING btree (id1, id2)"
+																	 )
+												)
+										)
+										.thenCompose( v -> s.createNativeQuery( indexDefinitionQuery, String.class )
+												.setParameter( 1, "aanother" )
+												.getSingleResult()
+												.thenAccept( result ->
+																	 context.assertEquals(
+																			 result,
+																			 "CREATE UNIQUE INDEX aanother_pkey ON public.aanother USING btree (id)"
```

Comment:
Not critical, but when using `context.equals(expected, actual)`, the firs value is the expected one and the second the actual one.

Reply:
I normally do it that way, I don't know why I didn't in this test. I'll change it.

Reply:
@DavideD, I pushed a [commit|https://github.com/hibernate/hibernate-reactive/pull/903/commits/99eb01a207506b1536aee6ca7ed4a2252db57632] that corrects the order.